### PR TITLE
Bump frameworks

### PR DESCRIPTION
**Pull request by Adamantios**

### CODE REVIEW

**Review by dvilelaf — COMMENTED** [review]
LGTM, but one question: do we have any news on the baseFee error that happens on Tenderly? Divya reported today that she is also getting it on Mode mainnet. I will need to test Contribute on Tenderly this week so I'm concerned about this bump blocking me.

**Comment by Adamantios** [comment]
> LGTM, but one question: do we have any news on the baseFee error that happens on Tenderly? Divya reported today that she is also getting it on Mode mainnet. I will need to test Contribute on Tenderly this week so I'm concerned about this bump blocking me.

No updates as there is not enough information to investigate this.
The error on Divya's side was caused by an incorrect transaction

**Comment by dvilelaf** [comment]
> > LGTM, but one question: do we have any news on the baseFee error that happens on Tenderly? Divya reported today that she is also getting it on Mode mainnet. I will need to test Contribute on Tenderly this week so I'm concerned about this bump blocking me.
> 
> No updates as there is not enough information to investigate this. The error on Divya's side was caused by an incorrect transaction and is resolved.

Then I dont think we should merge this yet

**Comment by Adamantios** [comment]
> > > LGTM, but one question: do we have any news on the baseFee error that happens on Tenderly? Divya reported today that she is also getting it on Mode mainnet. I will need to test Contribute on Tenderly this week so I'm concerned about this bump blocking me.
> > 
> > 
> > No updates as there is not enough information to investigate this. The error on Divya's side was caused by an incorrect transaction and is resolved.
> 
> Then I dont think we should merge this yet

Please run the branch and share full information on the issue you are facing. If this is a bug we won't merge.

File: packages/valory/agents/impact_evaluator/aea-config.yaml [hunk 1]
```diff
@@ -90,7 +90,7 @@ logging_config:
 skill_exception_policy: stop_and_exit
 dependencies:
   open-aea-ledger-ethereum:
-    version: ==1.53.0
+    version: ==1.58.0
```
Comment (dvilelaf) [inline packages/valory/agents/impact_evaluator/aea-config.yaml:93]: @Adamantios both agent and service yaml are missing some gas configuration in order to properly work

**Review by dvilelaf — APPROVED** [review]

File: packages/valory/agents/impact_evaluator/aea-config.yaml (hunk 1 above)
Comment (Adamantios) [reply packages/valory/agents/impact_evaluator/aea-config.yaml:93]: You will only need to override the value if you don't want to use dynamic gas pricing or if you want to use a testnet.

**Pull request merged**